packages/perseus/src/widgets/numeric-input/numeric-input.testdata.ts

```diff
@@ -36,6 +37,305 @@ export const question1: PerseusRenderer = {
     },
 };
 
+// Used for Storybook.
+export const defaultQuestion: PerseusRenderer = {
+    content: "$5008 \\div 4 =$ [[\u2603 numeric-input 1]] ",
+    images: {},
+    widgets: {
+        "numeric-input 1": {
+            graded: true,
+            version: {
+                major: 0,
+                minor: 0,
+            },
+            static: false,
+            type: "numeric-input",
+            options: {
+                coefficient: false,
+                static: false,
+                answers: [
+                    {
+                        status: "correct",
+                        maxError: null,
+                        strict: true,
+                        value: 1252,
+                        simplify: "required",
+                        message: "",
+                        // We don't include proper as otherwise
+                        // the tooltip will hide the examples.
+                        answerForms: [
+                            "integer",
+                            "mixed",
+                            "improper",
+                            "decimal",
+                            "pi",
+                        ],
+                    },
+                ],
+                answerForms: [
+                    // We don't include proper as otherwise
+                    // the tooltip will hide the examples.
+                    {simplify: "required", name: "integer"},
+                    {simplify: "required", name: "mixed"},
+                    {simplify: "required", name: "improper"},
+                    {simplify: "required", name: "decimal"},
+                    {simplify: "required", name: "pi"},
+                ],
```

Comment:
I don't think these are used, but I would love to know if you think otherwise (please DM me)